helpers/DateTimeHelper.js

```diff
@@ -17,9 +17,15 @@ function isWeekend (zonedDateTime) {
   return zonedDateTime.dayOfWeek().value() > 5
 }
 
+function reminderTime (timeToBeReminded) {
```

Comment:
Not a big fan of the name of this method, as it implies it can only be used for the reminders.
formatEpochSeconds better fits its purpose